### DISCUSSION TITLE
Feature request: Searchable configuration

### DISCUSSION THREAD

**Discussion by johanf85** [home-assistant/frontend#10003] [Accessibility]
Maybe already proposed by someone else (couldn't find it), but something that I would consider to come in hand is a search box on the configuration page, to easily find a setting. As I am lazy, this makes it a bit easier to find a certain setting rather than going trough options presented in the screen. 

Comparable to iOS searching settings:
![image](https://user-images.githubusercontent.com/43075793/132818509-6cbbafc8-655a-4d29-8cbd-b66af77cf8cb.png)

**Comment by ludeeus** [comment]
We already have that globally <https://www.home-assistant.io/docs/tools/quick-bar/>

**Reply by johanf85** [reply]
That seems to be something different than from what I meant. That is only entities and commands, not being able to search configuration.

**Reply by spacegaier** [reply]
Well yes and no: The quick-bar has "navigate" options to directly go to the sections in the config (automations, server control, ...). What we do have is a search for subsections (e.g. search or "elevation" and getting to the corresponding input field in the config.
